code/web/src/modules/user/api/actions.js

```diff
@@ -36,7 +36,7 @@ export function login(userCredentials, isLoading = true) {
     return axios.post(routeApi, query({
       operation: 'userLogin',
       variables: userCredentials,
-      fields: ['user {name, email, role}', 'token']
+      fields: ['user {name, email, role, bio, image, street, city, state, zip}', 'token']
```

Comment:
Thanks back end team this integration went super smoothly great work!

Reply:
Glad this all came together easily!